packages/desktop-client/src/components/table.tsx

```
@@ -393,27 +415,26 @@
+  function InputCell(
+    { inputProps, onUpdate, onBlur, textAlign, ...props }: InputCellProps,
+    ref,
+  ) {
+    return (
+      <Cell textAlign={textAlign} {...props} ref={ref}>
+        {() => (
+          <InputValue
+            value={props.value}
+            onUpdate={onUpdate}
+            onBlur={onBlur}
+            style={{ textAlign, ...(inputProps && inputProps.style) }}
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)